support-lib/jni/djinni_support.cpp

```diff
@@ -594,6 +598,8 @@ void jniDefaultSetPendingFromCurrentImpl(JNIEnv * env) {
         return;
     } catch (const std::exception & e) {
         env->ThrowNew(env->FindClass("java/lang/RuntimeException"), e.what());
+    } catch(...) {
```

Comment:
This doesn't look related to the purpose of this PR.  Can you explain?
I've generally learned to shy away from `catch(...)` since depending on the platform/compiler it can catch things you didn't intend, including things which aren't really C++ exceptions.

Reply:
Sorry, it is really not related. I revert this change now.